finance/management/commands/xformnetfilerawdata.py

```diff
@@ -273,7 +273,7 @@ def load_f460A_row(row, agency, verbosity=1):  # noqa
     beneficiary.save()
 
     # Now we have all the parts, save!
-    money = models.IndependentMoney(  # or calculated_Amount
+    money, _ = models.IndependentMoney.objects.get_or_create(  # or calculated_Amount
```

Comment:
I was not suggesting we don't have the unique constraint.  One reason to not just overwrite what is there would be if there were revisions in the data.  If there were then we would have to determine if this is a newer revision than what is stored.